common/lib/capa/capa/tests/test_responsetypes.py

```diff
@@ -2173,7 +2175,6 @@ def test_valid_xml(self):
         Test that `build_problem` builds valid xml
```

Comment:
It appears to test that the included factory creator method doesn't raise an exception... implicitly...